src/app/helpers/spies.ts

```diff
@@ -0,0 +1,21 @@
+export const resetSpies = (spies: any[]) => {
+    spies.forEach(spyObj => {
+      Object.keys(spyObj).forEach(prop => {
+        if (typeof spyObj[prop] === 'function') {
+          spyObj[prop].calls.reset();
+        }
+      });
+    });
+   };
+const createSpyObj = (name: string, methods: string[]) => {
+    return jasmine.createSpyObj(name, methods);
+};
+
+export const  loginServiceSpy = createSpyObj('LoginService', [
+    'login'
+]);
+
+export const  toastServiceSpy = createSpyObj('ToastrService', [
+    'success',
+    'error'
+]);
```

Comment:
@BrianSerem, We eliminated the blank line here, but we should keep it

Reply:
I have added an extra line here.